lib/puppet_x/cisco/cmnutils.rb

```diff
@@ -32,6 +32,38 @@ def self.process_network_mask(network)
         network
       end
 
+      # Helper utility for checking if arrays are overlapping in a
```

Comment:
Not blocking for this pull request - but we should really be writing RSpec tests for the methods in lib/puppet_x to demonstrate that they work as intended.